components/server/src/container-module.ts

```diff
@@ -206,8 +205,6 @@ export const productionContainerModule = new ContainerModule((bind, unbind, isBo
     bind(GitTokenScopeGuesser).toSelf().inSingletonScope();
     bind(GitTokenValidator).toSelf().inSingletonScope();
 
-    bind(BlockedUserService).to(BlockedUserService).inSingletonScope();
```

Comment:
Drive-by merges this service with the User-service. This was done to make stubbing out the UserService easier for tests